Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers_description.md

```diff
@@ -0,0 +1,34 @@
+## Configure Azure Risky Users on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for AzureRiskyUsers.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    |Client ID|True|
+    |Use system proxy|False|
+    |Trust any certificate|False|
+
+---
+
+To connect to the Azure Risky Users using either the Cortex XSOAR Azure application or the Self-Deployed Azure application:
```

Comment:
If "platform" isn't the best phrasing, let's choose another word, but it isn't a valid sentence at the moment.
Moreover, let's re-review the whole description and make sure it's perfect from a grammar and phrasing perspective
```suggestion
To connect to the Azure Risky Users platform using either the Cortex XSOAR Azure application or the Self-Deployed Azure application:
```